pkg/devconsole/proxy/proxy_test.go

```diff
@@ -0,0 +1,59 @@
+package proxy
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProxyEndpoint(t *testing.T) {
+	tests := []struct {
+		testName string
+		method   string
+		expected map[string]interface{}
+	}{
+		{
+			testName: "valid GET",
+			method:   "GET",
+			expected: map[string]interface{}{
+				"statusCode": http.StatusOK,
+			},
```

Comment:
Using types here allows you to assert also the headers and body later.

```suggestion
			request: {
			  Method: "GET", // you can also have a test for "" and "POST" maybe. ;)
			  Url: "testserver",
			},
			expectedResponse: {
				StatusCode: http.StatusOK,
				Headers: ...
				Body: "Mocked response",
			},
```